Dockerfile

```
@@ -0,0 +1,46 @@
+FROM golang:1.18-buster AS build-setup
+
+RUN apt-get update
+RUN apt-get -y install cmake zip sudo git
```

these should be run in the same layer, otherwise you could end up installing an old version unintentionally
```suggestion
RUN apt-get update
 && apt-get -y install cmake zip sudo git
```